### DISCUSSION TITLE
Plugin Class Names, Auto Height error when use with demo embla carousel thumbs

### DISCUSSION THREAD

**Discussion by sarussss** [davidjerleke/embla-carousel#625] [Help] [answered]
Hi @davidjerleke, I want to report an issue that I think is a bug.
I think the problem might be here slideRegistry. https://tppr.me/Cr2r https://tppr.me/4RTuT
Click thumb index 7 to see error.
Image: https://tppr.me/g7X5s  => https://tppr.me/KT4m7
Demo: https://codesandbox.io/s/embla-carousel-thumbs-vanilla-forked-2cywpr?file=/src/js/index.js

**Comment by davidjerleke** [comment]
Thank you @sarussss, I will look into it when possible.

Best,
David

**Reply by davidjerleke** [reply, marked as answer]
@sarussss I can confirm that this is a bug. It's caused by the `SlideRegistry` component in the core package `embla-carousel`. When `containScroll: 'keepSnaps'` is set, it groups slides in the registry but it shouldn't. Thanks for reporting this 👍. 

- I'm closing this discussion because this bug is now tracked in #629.

Best,
David

**Reply by sarussss** [reply]
Thank you @davidjerleke.
